beacon-chain/cache/depositcache/deposits_cache_test.go

```diff
@@ -409,6 +409,16 @@ func TestFinalizedDeposits_UtilizesPreviouslyCachedDeposits(t *testing.T) {
 	}
 }
 
+func TestFinalizedDeposits_DepositsAreNil_ReturnsNil(t *testing.T) {
+	dc := DepositCache{}
+
+	deposits := dc.finalizedDeposits
```

Comment:
```suggestion
	deposits := dc.FinalizedDeposits()
```

Reply:
`finalizedDeposits`  is not a function. Is there something that I'm missing here?

Reply:
You are testing the function `FinalizedDeposits(ctx context.Context)`, that's entry point to the panic

Reply:
Uhhh... Gotta be more careful next time. Thanks!